wolox-books/src/app/screens/register/register.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="register">
+  <img class="register__logo" src="/assets/images/logo-wolox.png" alt="Logo Wolox">
+  <form [formGroup]="registerForm" (ngSubmit)="registerUser(registerForm.value)" class="register__form">
+    <div class="register__name form-input">
+      <label class="form-input__label" for="name">Nombre</label>
+      <input class="form-input__input" name="name" id="name" formControlName="name">
+    </div>
+    <div class="register__last-name form-input">
+      <label class="form-input__label" for="last-name">Apellido</label>
+      <input class="form-input__input" name="last-name" id="last-name" formControlName="last-name">
+    </div>
+    <div class="register__email form-input">
+      <label class="form-input__label" for="email">Email</label>
+      <input class="form-input__input" type="email" name="email" id="email" formControlName="email">
+      <span class="form-input__error" *ngIf="registerForm.controls['email']?.errors?.required && registerForm.controls['email']?.touched">El email es requerido</span>
```

Comment:
create variable in controller for forms controls and use them here, example:
`this.emailControl = this.registerForm.controls['email'] as FormControl;`

```suggestion
      <span class="form-input__error" *ngIf="emailControl.hasError('required')">
        El email es requerido
      </span>
```

Reply:
Getter created to email form control and changed to hasError() method

Reply:
Getter created to password and password_confirmation too.